cve-list.html

```diff
@@ -9,6 +9,63 @@ <h1>Apache Kafka Security Vulnerabilities</h1>
 
 This page lists all security vulnerabilities fixed in released versions of Apache Kafka.
 
+<h2><a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44228">CVE-2021-44228</a>
+  Flaw in Apache Log4j logging library in versions from 2.0.0 and before 2.15.0</h2>
+
+  <p>Some components in Apache Kafka use <code>Log4j-v1.2.17</code>  there is no dependence on <code>Log4j v2.*</code></p>
+  
+  <p><a href="https://logging.apache.org/log4j/2.x/manual/lookups.html">Lookups feature</a> was introduced in Log4j v2.x in order to allow specifying Log4j configuration parameters in arbitrary locations (even outside of the configuration files). Log4j v1.x does not offer the same functionality and thus is not vulnerable to <a href="https://access.redhat.com/security/cve/cve-2021-44228">CVE-2021-44228</a>.</p>
+  <p>Users should NOT be impacted by this vulnerability</p>
+  
+  <table class="data-table">
+  <tbody>
+    <tr>
+      <td>Versions affected</td>
+      <td>NA</td>
+    </tr>
+    <tr>
+      <td>Fixed versions</td>
+      <td>NA</td>
+    </tr>
+    <tr>
+      <td>Impact</td>
+      <td>NA</td>
+    </tr>
+    <tr>
+      <td>Issue announced</td>
+      <td>09 Dec 2021</td>
+    </tr>
+  </tbody>
+  </table>
+
+<h2><a href="https://access.redhat.com/security/cve/CVE-2021-4104">CVE-2021-4104</a>
+  Flaw in Apache Log4j logging library in versions 1.x</h2>
+  
+  <p>Some components in Apache Kafka use <code>Log4j-v1.2.17</code></p>
```

Comment:
Could we spell out all components? I think the components include the following.

broker
controller
zookeeper
connect
mirrormaker
tools
clients configured with log4j

Reply:
How does something like this sound?

"Some components, including but not limited to: broker, controller, zookeeper, connect, mirrormaker, tools, and clients configured with log4j, use `Log4j-v1.2.17`"

Reply:
How about the following?

"The following components in Apache Kafka use Log4j-v1.2.17: broker, controller, zookeeper, connect, mirrormaker and tools. Clients may also be configured to use Log4j-v1.2.17."

Reply:
Here it is with both updates:
<img width="1024" alt="Screen Shot 2021-12-14 at 1 57 20 PM" src="https://user-images.githubusercontent.com/66280178/146062124-bc90d370-91ed-40e7-a0c6-771c0bf9d856.png">

Reply:
A minor change. "Client applications may also be configured to use Log4j-v1.x."

Reply:
Is this to update the last sentence of the first paragraph?